source/robot_model/include/robot_model/Model.hpp

```diff
@@ -211,6 +225,7 @@ inline const std::vector<std::string>& Model::get_joint_names() const {
 }
 
 inline unsigned int Model::get_nb_joints() const {
+  // this is technically not correct
```

Comment:
I think we should be precise and take model.njoints - 1. model.nq/model.nv have slightly different meanings, see https://gepettoweb.laas.fr/doc/stack-of-tasks/pinocchio/master/doxygen-html/structpinocchio_1_1ModelTpl.html